src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+import java.util.ArrayList;
+/**
+ * Tagging Association class for associating Person and Tag.
+ * @author CHUAN
+ *
+ */
+public class Tagging {
+	private Person person;
+	private Tag tag;
+	private String tagOperation;
```

Comment:
Using String is a risky choice to keep track of the addition/deletion. Very hard to hunt down mistakes like accidentally mispell the operation e.g. "Deleet" instead "Delete". 

You can consider the alternative of defining an enumeration for OperationType (e.g.) with "ADD" and "DELETE" as values.